.github/workflows/actions.yml

```diff
@@ -25,4 +29,20 @@ jobs:
       run: |
         npm ci
         npm test
+  lint:
+    name: Langium Lint
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Use Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: '16'
+    - name: Build
+      shell: bash
+      run: |
+        npm ci
         npm run lint
```

Comment:
The build is probably faster if we keep the linting step in the matrix job?

Reply:
For some reason, running `npm run lint` on Windows GitHub Actions runs into some sort of freeze. The CI times out after 20 minutes, see [here](https://github.com/langium/langium/actions/runs/3812281182/jobs/6485406094). That's why I split them. It's not necessary to run the lint twice.

The build is actually faster if we don't, since these actions run in parallel not sequentially.

Reply:
Also, it might make sense to split tests and linting anyway, we have done the same in Theia recently. Contributors don't want to see every pipeline fail just because of an extra whitespace somewhere. Just failing the lint pipeline is good enough.